### PR TITLE
Started working on fixing/adding examples

### PR DESCRIPTION
**Pull request by mattatcha**
My goal is to include all examples from the rethinkdb cookbook. Some examples were already created but they had to be renamed for godoc.
#27

### CODE REVIEW

File: query_write.go [hunk 1]
```diff
@@ -20,9 +20,6 @@ func (o *InsertOpts) toMap() map[string]interface{} {
 // 'hard' or 'soft', to override the table or query's default durability setting,
 // or the optional argument return_changes, which will return the value of the row
 // you're inserting when set to true.
-//
-//	table.Insert(map[string]interface{}{"name": "Joe", "email": "joe@example.com"}).RunWrite(sess)
-//	table.Insert([]interface{}{map[string]interface{}{"name": "Joe"}, map[string]interface{}{"name": "Paul"}}).RunWrite(sess)
 func (t Term) Insert(arg interface{}, optArgs ...InsertOpts) Term {
```
Comment (mattatcha) [inline query_write.go:23]: Removed in favor of insert examples.

**Comment by dancannon** [comment]
Thanks for getting started on this, once I have finished fixing all of the comments I will start helping out. 

I also think it might be worth creating some fixtures that are run before any of the examples but I am not quite sure what data needs to be setup yet (maybe something based off of the RethinkDB docs?).

**Comment by mattatcha** [comment]
No problem.

I don't know for sure. I will have to check the cookbook.

**Comment by dancannon** [comment]
:+1: My reasoning for wanting to setup fixtures is that otherwise each example will have to have quite a bit of boilerplate code for setting up DBs+Tables+data. I have done this for the tests and it gets pretty messy/unmaintainable.

If we get something good working for the examples I might migrate all the tests as well.

**Pull request merged**